interface/github.py

```diff
@@ -111,7 +112,9 @@ def org_get_teams(self):
         team_array = []
         for team in teams:
             # convert PaginatedList to List
-            team_array.append(team)
+            team_model = ModelTeam(team.id, team.name, "")
+            team_model.members = set(self.list_team_members(team.id))
```

Comment:
I should probably add type annotations to `interface/*.py` files, but...

`list_team_members` returns a list of the team members, not the member's github ids. You would probably need to change the function to return that, or select it using a map lambda.

Reply:
ooooo okok

Reply:
would something like this work? 
```suggestion
            team_model.members = set(user.id
                                     for user in
                                     self.list_team_members(team.id))
```

Reply:
Yes. Most likely.